packages/core/review-workflows/admin/src/components/Widgets.tsx

```diff
@@ -0,0 +1,134 @@
+        {documents?.map((document) => (
+          <Tr onClick={handleRowClick(document)} cursor="pointer" key={document.documentId}>
+            <Td>
+              <CellTypography title={document.title} variant="omega" textColor="neutral800">
+                {document.title}
+              </CellTypography>
+            </Td>
+            <Td>
+              <CellTypography variant="omega" textColor="neutral600">
+                {document.kind === 'singleType'
+                  ? formatMessage({
+                      id: 'content-manager.widget.last-edited.single-type',
+                      defaultMessage: 'Single-Type',
+                    })
+                  : formatMessage({
+                      id: document.contentTypeDisplayName,
+                      defaultMessage: document.contentTypeDisplayName,
+                    })}
+              </CellTypography>
+            </Td>
+            <Td>
+              <Box display="inline-block">
+                {document.status ? (
+                  <DocumentStatus status={document.status} />
+                ) : (
+                  <Typography textColor="neutral600" aria-hidden>
+                    -
+                  </Typography>
+                )}
+              </Box>
+            </Td>
+            <Td>
+              <Typography textColor="neutral600">
+                <RelativeTime timestamp={new Date(document.updatedAt)} />
+              </Typography>
+            </Td>
+            <Td>
+              <StageColumn strapi_stage={document.strapi_stage} />
+            </Td>
+            <Td onClick={(e) => e.stopPropagation()}>
+              <Box display="inline-block">
+                <IconButton
+                  tag={Link}
+                  to={getEditViewLink(document)}
+                  onClick={() => trackUsage('willEditEntryFromHome')}
+                  label={formatMessage({
+                    id: 'content-manager.actions.edit.label',
+                    defaultMessage: 'Edit',
+                  })}
+                  variant="ghost"
+                >
+                  <Pencil />
+                </IconButton>
+              </Box>
+            </Td>
+          </Tr>
+        ))}
+      </Tbody>
+    </Table>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * AssignedWidget
+ * -----------------------------------------------------------------------------------------------*/
+
+const AssignedWidget = () => {
+  const { formatMessage } = useIntl();
+  const { data, isLoading, error } = useGetRecentlyAssignedDocumentsQuery();
+
+  if (isLoading) {
+    return <Widget.Loading />;
+  }
+
+  if (error || !data) {
+    return <Widget.Error />;
+  }
+
+  if (data.length === 0) {
+    return (
+      <Widget.NoData>
+        {formatMessage({
+          id: 'content-manager.widget.last-published.no-data',
+          defaultMessage: 'No entries',
+        })}
+      </Widget.NoData>
+    );
```

Comment:
not sure how but I end up with "no published entries instead" 🤔

<img width="1352" height="730" alt="Image" src="https://github.com/user-attachments/assets/25878860-88c0-4060-b615-42402dda0778" />